config/webpack.config.js

```diff
@@ -236,9 +241,14 @@ module.exports = env => {
       }),
 
       new MiniCssExtractPlugin({
-        filename: !isOptimizedBuild
-          ? '[name].css'
-          : `[name].[contenthash]-${timestamp}.css`,
+        moduleFilename: ({ name, entryModule }) => {
```

Comment:
`moduleFilename` docs: https://webpack.js.org/plugins/mini-css-extract-plugin/#module-filename-option

Reply:
Here are the keys that are on the object argument for the `moduleFilename` function:
```
[ 'id',
  'ids',
  'debugId',
  'name',
  'preventIntegration',
  'entryModule',
  '_modules',
  'filenameTemplate',
  '_groups',
  'files',
  'rendered',
  'hash',
  'contentHash',
  'renderedHash',
  'chunkReason',
  'extraAsync',
  'removedModules' 
]
```

Reply:
https://webpack.js.org/configuration/output/#template-strings

![image](https://user-images.githubusercontent.com/6130520/94280978-ae2f0500-ff13-11ea-9d1c-8ec9f736fc70.png)